packages/build-tools/src/steps/functions/sendSlackMessage.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  BuildFunction,
+  BuildStepContext,
+  BuildStepInput,
+  BuildStepInputValueTypeName,
+} from '@expo/steps';
+import { BuildStepInputById } from '@expo/steps/dist_esm/BuildStepInput';
+
+export function createSendSlackMessageFunction(): BuildFunction {
+  return new BuildFunction({
+    namespace: 'eas',
+    id: 'send_slack_message',
+    name: 'Send Slack message',
+    inputProviders: [
+      BuildStepInput.createProvider({
+        id: 'slack_hook_url',
+        allowedValueTypeName: BuildStepInputValueTypeName.STRING,
+        required: true,
+      }),
```

Comment:
I think that currently AFAIK you can't reference secrets in function inputs 🤔 

What if somebody considers it sensitive data and doesn't want to put it as plain text in the config file? 

I'm suggesting here that maybe we should also read this value from `SLACK_WEBHOOK_URL` env var/secret?

Reply:
That may be a good idea

Reply:
Changed plain input into a secret